pkg/gitreceive/k8s_util_test.go

```diff
@@ -94,7 +98,10 @@ func TestBuildPod(t *testing.T) {
 
 		checkForEnv(t, pod, "TAR_PATH", build.tarKey)
 		checkForEnv(t, pod, "PUT_PATH", build.putKey)
-		checkForEnv(t, pod, "CACHE_PATH", build.cacheKey)
+
+		if _, ok := build.env["DEIS_DISABLE_CACHE"]; !ok {
+			checkForEnv(t, pod, "CACHE_PATH", build.cacheKey)
+		}
```

Comment:
Could we possibly additionally assert that `CACHE_PATH` is not set when `DEIS_DISABLE_CACHE` is set?
Maybe something like:

```go
else {
    if _, err := envValueFromKey(pod, "CACHE_PATH") (string, error); err == nil {
        ...
    }
}`